java/src/main/java/com/saucelabs/saucebindings/SauceSession.java

```diff
@@ -20,6 +21,10 @@ public SauceSession() {
         this(new SauceOptions());
     }
 
+    public SauceSession(SauceConfigs configs) {
+        this(configs.build());
```

Comment:
This is kind of a hack in case people "do it wrong." We want them to build it themselves and pass it in, but we were concerned that they would get a weird error message if they didn't, and it didn't seem that hard to just add a constructor to make it just work.